examples/efs/mounttarget.yaml

```diff
@@ -38,4 +41,17 @@ spec:
     vpcIdSelector:
       matchLabels:
         testing.upbound.io/example-name: efs
-    cidrBlock: 172.16.10.0/24
\ No newline at end of file
+    cidrBlock: 172.16.10.0/24
+---
+apiVersion: ec2.aws.upbound.io/v1beta1
+kind: SecurityGroup
+metadata:
```

Comment:
Could you please add the `meta.upbound.io/example-id` annotation to both the dependent & dependency resource manifests?

Reply:
do we want to include just two resources or the whole dependency chain(down to VPC)?

Reply:
@ulucinar i've added the metadata to the whole example